### DISCUSSION TITLE
End of Audio/video to text notification

### DISCUSSION THREAD

**Discussion by devkatalist** [SubtitleEdit/subtitleedit#6044] [Ideas]
Hello. Sometimes, the Audio/video to text option takes a lot of time (especially with a batch), so we leave it in the background to do other tasks. It would be handy to have some sort of sound notification (or other) to alert us that it is done. What do you think?

**Comment by niksedk** [comment]
The taskbar icon could be a progress bar - would that help?

**Reply by devkatalist** [reply]
Thank you for your answer @niksedk, that would be helpful for me, yes. But I wonder if it will be sufficient for other users.

**Reply by niksedk** [reply]
OK, how is this: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.6/SubtitleEditBeta.zip
It should display a progress bar in the status bar - and blink/beep if SE is not focused when finished.

**Reply by devkatalist** [reply]
Sounds perfect! Many thanks from all of us!
